scripts/modules/validation/sanitizers.js

```
@@ -0,0 +1,667 @@
+				modelConfig.maxTokens < 1
+			) {
+				log('warn', `Invalid maxTokens for ${role}, using default`);
+				delete modelConfig.maxTokens;
+			}
+		}
+
+		if (modelConfig.temperature !== undefined) {
+			modelConfig.temperature = Number(modelConfig.temperature);
+			if (!Number.isFinite(modelConfig.temperature)) {
+				log('warn', `Invalid temperature for ${role}, using default`);
+				delete modelConfig.temperature;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)